maxim/tests/test_test_runs.py

```
@@ -241,5 +263,597 @@ def error(self, message: str, e: Optional[Exception] = None) -> None:
         if result is not None:
             self.assertListEqual(result.failed_entry_indices, [])
 
+    def test_create_test_run_with_image_variables_and_prompt_workflow(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_create_test_run_with_image_variables_and_prompt_workflow`

Add return type annotation: `None`

(ANN201)